go/runtime/host/helpers.go

```diff
@@ -22,6 +22,11 @@ var (
 	ErrInternal = fmt.Errorf("runtime: internal error")
 )
 
+const (
+	// MethodQueryBatchWeightLimits is the name of the runtime batch weight limits query method.
+	MethodQueryBatchWeightLimits = "internal.BatchWeightLimits"
```

Comment:
Should this be moved to `go/runtime/host/protocol` as it is kind of part of the RHP?